src/main/java/net/geforcemods/securitycraft/entity/SentryEntity.java

```diff
@@ -219,16 +246,29 @@ else if(item == SCContent.UNIVERSAL_OWNER_CHANGER.get())
 	}
 
 	/**
-	 * Cleanly removes this sentry from the world, dropping the module and removing any block
+	 * Cleanly removes this sentry from the world, dropping the module and removing the block the sentry is disguised with
 	 */
 	@Override
 	public void remove()
 	{
+		BlockPos pos = getPosition();

```

Comment:
1. New line after this.
2. You added this variable, why did you not replace all occurances of `getPosition()` in this method?